attesters/hydra-s1/index.ts

```diff
@@ -1,19 +1,26 @@
-import { AttestationsCollection } from "../../src/topics/attestations-collection/attestations-collection";
+import { AttestationsCollection } from "../../src/topics/attestations-collection";
+import { Attester, AttesterNetwork } from "../../src/topics/attester";
 import { Badge } from "../../src/topics/badge";
-import { Attester } from "../../src/topics/attester";
 import { Group } from "../../src/topics/group";
 
-export const hydraS1SimpleAttester = async () =>
+export default async () =>
   new Attester({
-    collectionIdFirst: 10000001,
+    name: "hydra-s1-simple",
+    configurations: {
+      polygon: {
+        firstCollectionId: 10000001,
+        address: "",
+      },
+    },
+    defaultCurrentTargetNetwork: AttesterNetwork.Polygon,
```

Comment:
I don't understand the need for "defaultCurrentTargetNetwork" ? 
What would be the default if the attester is deployed on Mainnet, Polygon and Optimism?

Reply:
I changed it by using a getter in the attester that throws when no currentTargetNetwork is set